README.md

```diff
@@ -59,7 +59,7 @@ Flags:
                           default - default section, contains all rest imports
                           blank - blank section, contains all blank imports. This section is not presed unless explicitly enabled. (default [standard,default])
       --skip-generated    Skip generated files
-
+      --custom-order      Enable custom order of sections
```

Comment:
Please add default comment.

Reply:
Please help. Is this comment enough?
`--custom-order      Enable custom order of sections. The default order is standard: standard > default > custom > blank > dot.`

Reply:
`If specified, make Make the section order the same as your configuration order. The default order is standard > default > custom > blank > dot.`

Reply:
Thanks. Done